addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -182,13 +182,7 @@ models.Order = models.Order.extend({
             var qty  = Number(line.get_quantity());
             var note = line.get_note();
             var product_id = line.get_product().id;
-
-            if (typeof resume[line_hash] === 'undefined') {
-                resume[line_hash] = { qty: qty, note: note, product_id: product_id };
-            } else {
-                resume[line_hash].qty += qty;
-            }
-
+            resume.push({ qty: qty, note: note, product_id: product_id });
```

Comment:
This used to consolidate orderlines with the same `line_hash`. By pushing to an array you remove that behavior, which I don't think is the intention of the task. I suppose the best way to fix this issue is by turning the object into an array, but you should still consolidate if you can. You could put the line_hash as a property of every element in the array, and then loop through it before you push to see if you can't consolidate first.